ccp/modules/datashield-setup.sh

```diff
@@ -12,4 +12,12 @@ if [ "$ENABLE_DATASHIELD" == true ]; then
     chown -R bridgehead:docker /tmp/bridgehead/
     chmod g+r /tmp/bridgehead/opal-key.pem
   fi
+  mkdir -p /tmp/bridgehead/opal-map
+  jq -n --argfile input ./$PROJECT/modules/datashield-mappings.json '
+    {
+        "external": "opal-'"$SITE_ID"'",
+        "internal": "opal:8080",
+        "allowed": [$input.sites[].id | "datashield-connect.\(.).broker.ccp-it.dktk.dkfz.de"]
+    }' > /tmp/bridgehead/opal-map/local.json
+  cp ./$PROJECT/modules/datashield-mappings.json /tmp/bridgehead/opal-map/central.json
```

Comment:
1. I dont think it will? cp overrides stuff by default no?
2. It gets copied to this directory because this is the directory that gets mounted to beam-connect. I know we could mount this file and the local file to not have to copy but I think this is cleaner and this is not a gigantic file so I think its fine